reactioncommerce/reaction-docs
# docs: Operator Guide for new Orders panel

Resolves #792   
Impact: **minor**  
Type: **docs**

Update Operator guide with new Orders UI

## Testing
1.Start the docs
1. Navigate to the "Next" version
1. See the `Operators guide - Orders` section is updated
